### PR TITLE
Feature/omg 245 run tests on postgres database

### PR DESCRIPTION
**Pull request by purbanow**

### CODE REVIEW

File: apps/omg_watcher/config/dev.exs [hunk 1]
```diff
@@ -41,13 +41,8 @@ config :phoenix, :stacktrace_depth, 20
 # Configure your database
 config :omg_watcher, OMG.Watcher.DB.Repo,
   adapter: Ecto.Adapters.Postgres,
-  username: "omisego_dev",
-  password: "omisego_dev",
-  database: "omisego_dev",
-  hostname: "localhost",
-  pool_size: 10,
-  timeout: 60_000,
-  connect_timeout: 60_000
```
Comment (pnowosie) [inline apps/omg_watcher/config/dev.exs]: revert ```  pool_size: 10,
  timeout: 60_000,
  connect_timeout: 60_000```

File: apps/omg_watcher/config/dev.exs [hunk 2]
```diff
@@ -41,13 +41,8 @@ config :phoenix, :stacktrace_depth, 20
...
+  pool: Ecto.Adapters.SQL.Sandbox,
+  url: {:system, "DATABASE_URL", "postgres://omisego_dev:omisego_dev@localhost/omisego_dev"}
```
Comment (pnowosie) [inline apps/omg_watcher/config/dev.exs]: Pls find web resource and conform to the naming

File: apps/omg_watcher/config/prod.exs [hunk 3]
```diff
@@ -18,9 +18,11 @@ config :omg_watcher, OMG.Watcher.Web.Endpoint,
   url: [host: "example.com", port: 80],
   cache_static_manifest: "priv/static/cache_manifest.json"
 
+# Configure your database
 config :omg_watcher, OMG.Watcher.DB.Repo,
-  load_from_system_env: true,
-  adapter: Ecto.Adapters.Postgres
+  adapter: Ecto.Adapters.Postgres,
+  pool: Ecto.Adapters.SQL.Sandbox,
```
Comment (pnowosie) [inline apps/omg_watcher/config/prod.exs]: Ecto.Adapters.SQL.Sandbox, pls only for test

File: apps/omg_watcher/config/dev.exs (hunk 2 above)
Comment (purbanow) [reply apps/omg_watcher/config/dev.exs]: I added the following comment `# DATABASE_URL format is following `postgres://{user_name}:{password}@localhost/{database_name}``

File: apps/omg_watcher/config/test.exs [hunk 4]
```diff
@@ -6,12 +6,11 @@ config :omg_watcher, OMG.Watcher.Web.Endpoint,
...
 
-# Configure your database
 config :omg_watcher, OMG.Watcher.DB.Repo,
-  adapter: Sqlite.Ecto2,
-  database: "/tmp/omisego/ecto_simple_" <> Integer.to_string(:rand.uniform(10_000_000)) <> ".sqlite3"
-
-config :omg_watcher, OMG.Watcher.DB.Repo, pool: Ecto.Adapters.SQL.Sandbox
+  adapter: Ecto.Adapters.Postgres,
+  pool: Ecto.Adapters.SQL.Sandbox,
+  # DATABASE_URL format is following `postgres://{user_name}:{password}@localhost/{database_name}`
```
Comment (pnowosie) [inline apps/omg_watcher/config/test.exs]: I think it's `postgres://{user_name}:{password}@{machine}:{port}/{database_name}`

Comment (purbanow) [reply apps/omg_watcher/config/test.exs]: Yes, thanks!

**Pull request merged**